src/instructlab/model/backends/backends.py

```diff
@@ -39,6 +40,10 @@
 API_ROOT_WELCOME_MESSAGE = "Hello from InstructLab! Visit us at https://instructlab.ai"
 CHAT_TEMPLATE_AUTO = "auto"
 CHAT_TEMPLATE_TOKENIZER = "tokenizer"
+
+# Internal env variable for CI adjustment / disablement
+MAX_STABLE_VRAM_WAIT = int(os.getenv("ILAB_MAX_STABLE_VRAM_WAIT", "30"))
```

Comment:
this will raise on import. I think we will have to handle `ValueError`s when an invalid value is passed. I'd suggest to move the type coercion from import time to runtime, perhaps introduce a separate helper function called `get_vram_stable_wait_timeout()` and hide the error handling there.